src/models/chocolate_rand_forest.py

```diff
@@ -0,0 +1,65 @@
+# author: Manvir Kohli, Julie Song, Kelvin Wong
+# date: 2022-11-26
+
+"""This script creates a Random Forest model using the preprocessed input
+features from the chocolate exploration dataset. It dumps a tuned random
+forest model.
+
+Usage: src/models/chocolate_rand_forest.py --train=<training_ds> --output=<output_folder> --output-cv=<output_cv_folder>
+
+Options:
+--train=<training_ds>          Path to the training dataset, which is a .csv file
+--output=<output_folder>       Path to the folder to save the modelling output to
+--output-cv=<output_cv_folder> Path to the folder to save the CV score files to
+"""
+
+from docopt import docopt
+from scipy.stats import randint
+from sklearn.pipeline import make_pipeline
+from sklearn.ensemble import RandomForestRegressor
+
+from ..preprocessor.chocolate import preprocessor
+from .base_chocolate_model_tuner import BaseChocolateModelTuner
+
+
+class ChocolateDecisionTreeTuner(BaseChocolateModelTuner):
```

Comment:
Consider renaming this tuner class as, say, `ChocolateRandForestTuner`?

Reply:
Good catch! Will fix that.